src/authorization-control.ts

```diff
@@ -274,4 +275,47 @@ export class AuthorizationControlApi {
     }
     return { success: !resp?.payload?.errorCode, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take an Authorization Control Merchant Identifier Exemption
+   * token, and an active status argument, send that to Marqeta, and have an
+   * Authorization Control Merchant Identifier Exemption's active status updated.
+   * If not active status argument is supplied, the default active status is set
+   * to false.
+   */
+  async updateMerchantStatus(merchant: Partial<Merchant>): Promise<{
+    success: boolean,
+    body?: Merchant,
+    error?: MarqetaError,
+  }> {
+    const {
+      token,
+      active = false,
+    } = merchant
+    if (!token) {
+      return {
+        success: true,
+        body: {
+          name: merchant.name ?? '',
+        }
+      }
+    }
+    const resp = await this.client.fire('PUT',
+      `authcontrols/exemptmids/${token}`,
+      undefined,
+      { active: active },
+    )
```

Comment:
OK... why not simplify this to:

```typescript
     const resp = await this.client.fire('PUT',
       `authcontrols/exemptmids/${token}`,
       undefined,
       { active },
     )
```

as `active` is what you want it to be in the body of this `fire()`?